.github/workflows/_linux-test.yml

```diff
@@ -120,6 +97,16 @@ jobs:
         id: parse-ref
         run: .github/scripts/parse_ref.py
 
+      - name: Check for keep-going label
+        # This uses the filter-test-configs action because it conviniently
+        # checks for labels.  It does not actually do any filtering.  All
+        # filtering is done in the build step.
+        id: keep-going
+        uses: ./.github/actions/filter-test-configs
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This would increase the token usage here.  No action needed, but let's keep an eye out if there is a rate limit (A potential solution is to continue on error while assuming keep-going would be false by default)